js/ui/tab_panel.js

```diff
@@ -381,7 +381,6 @@ var TabPanel = MultiView.inherit({
                 var newItem = value ? this._tabs._itemElements().eq(id) : value;
                 this._setTabsOption("focusedElement", getPublicElement(newItem));
                 this.callBase(args);
-                this._tabs.focus();
```

Comment:
Why have we used the native focusing for TabPanel?

Reply:
This line had been added by @Aden-git in 15.1 (5e7bd8569386) to focus tabs. Currently, tabs get focus correctly